### ISSUE TITLE
[Accessibility] - Narrator + New Edge. Deleted text is not announced.

### ISSUE THREAD

**Issue by jswiderski** [ckeditor/ckeditor4#5165]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. Open CKEditor sample in new Edge
2. Open Narrator 
3. Try typing some text - Narrator announces typed text
4. Try deleting some text

### Expected result

Narrator announces deleted letters.

### Actual result

Narrator doesn't announce deleted letters.

## Other details

I have tried native contenteditable <div> as well as native contenteditable <iframe> - in both cases deleted text is announced.

* Browser:New Edge
* OS: Windows
* CKEditor version: 4.18.0
* Installed CKEditor plugins: N/A

**Comment by Comandeer** [comment]
I can confirm the issue. However, it seems to become visible only after moving the selection and then trying to delete characters – like the selection "lags" behind.

**Comment by jacekbogdanski** [comment]
It is likely that these issues are caused by the fault of the Narrator software, not CKEditor 4. We are officially using the latest Firefox version with the latest JAWS as our reference environment and officially supported platform for accessibility software ([see documentation](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_a11y.html#supported-platforms)). Therefore, these issues are not on our priority list at the moment as they require a significant commitment to the research.
